.travis.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+CONF_CRYPTO=""
+CONF_OBJSTORE=""
+
+case $CRYPTO in
+botan)
+	CONF_CRYPTO="$CONF_CRYPTO --with-crypto-backend=botan --with-botan=/usr"
+	CONF_CRYPTO="$CONF_CRYPTO --disable-ecc --disable-gost"
+	;;
+openssl)
+	CONF_CRYPTO="$CONF_CRYPTO --with-crypto-backend=openssl --with-openssl=/usr"
+	CONF_CRYPTO="$CONF_CRYPTO --disable-gost"
+	openssl version -a
+	;;
+*)
+	echo "Unknown crypto backend"
+	exit 1
+esac
+
+case $OBJSTORE in
```

Comment:
Maybe we do not need to build with and without "--with-objectstore-backend-db". The file backend will always be supported. It is not like OpenSSL or Botan, where you only have one backend in the build.

Perhaps we always can make the test build with "--with-objectstore-backend-db --with-migrate"? Or do we want to test a build without SQLite support?